File: programmers/코딩테스트 고득점 Kit/정렬/K번째수.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

vector<int> solution(vector<int> array, vector<vector<int>> commands) {
	vector<int> answer;
	for (int i = 0; i < commands.size(); i++) {
		vector<int> sortv;
		int s = commands[i][0];
		int e = commands[i][1];
		int k = commands[i][2];
		for (int j = s - 1; j < e; j++) sortv.push_back(array[j]);
		sort(sortv.begin(), sortv.end());
		answer.push_back(sortv[k - 1]);
	}
	return answer;
}

File: programmers/코딩테스트 고득점 Kit/정렬/K번째수.java
import java.util.*;
class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int[] answer = new int[commands.length];
        for(int i=0;i<commands.length;i++) {
            ArrayList<Integer> arr = new ArrayList<>();
            int s = commands[i][0];
            int e = commands[i][1];
            int rank = commands[i][2];
            for(int j= s - 1;j < e; j++) arr.add(array[j]);
            Collections.sort(arr);
            answer[i] = arr.get(rank-1);
        }
        return answer;
    }
}